cap/config.py

```diff
@@ -167,6 +165,22 @@ def _(x):
     ),
 )
 
+#: Record search facets.
+RECORDS_REST_FACETS = {
```

Comment:
Move over line 121 as it overrides the dict update for DEPOSIT_REST_FACETS (see https://github.com/JavierDelgadoFernandez/analysis-preservation.cern.ch/blob/8523cb3ff9a57aed9a5b532b46cba9e3f629a270/cap/config.py#L121)